### DISCUSSION TITLE
Add `borg info` on archive to Vorta GUI

### DISCUSSION THREAD

**Discussion by archont00** [borgbase/vorta#828] [Ideas]
**Is your feature request related to a problem? Please describe.**
Provide user with information on archive size (increment).
```
borg info ::archive-name
Archive name: archive-name
Archive fingerprint: b969c9dfblablablabfaaa413c1
Comment: 
Hostname: nothere
Username: root
Time (start): Sun, 2020-11-01 17:44:56
Time (end): Sun, 2020-11-01 17:47:47
Duration: 2 minutes 50.41 seconds
Number of files: 28821
Command line: /usr/bin/borg create --patterns-from /etc/borgmatic/default.patterns --patterns-from /etc/borgmatic/host.patterns --exclude-caches --exclude-if-present .nobackup --keep-exclude-tags --compression auto,zstd,10 --lock-wait 60 --info --stats --list --filter AME 'root@example.com:/path/to/repo::{hostname}-{now:%Y-%m-%dT%H:%M}' /
Utilization of maximum supported archive size: 0%
------------------------------------------------------------------------------
                       Original size      Compressed size    Deduplicated size
This archive:                5.10 GB              4.13 GB             20.04 MB
All archives:               20.40 GB             16.51 GB              4.17 GB

                       Unique chunks         Total chunks
Chunk index:                   26573               117353
```
I find the "Deduplicated size for this archive" useful.

**Describe the solution you'd like**
Add a button `Info` to "Archives" tab next to the existing ones under the archives list.

**Describe alternatives you've considered**
Let user right-click on archive name and run an arbitrary borg command with output displayed in a simple text window.

**Comment by samuel-w** [comment]
The issue is that getting the size of each repository is very slow, so we only get the archive size after it was made.

https://github.com/borgbase/vorta/issues/341#issuecomment-635275972

**Comment by archont00** [comment]
Indeed, the speed of getting info for all archives is too slow.

How about getting info on a particular archive only as I suggested? Maybe it could be a more generic request: the user could define arbitrary commands in Vorta prefs and right-click on archive name would allow to apply these commands and display the full output.

**Comment by samuel-w** [comment]
That would violate the intent of Vorta, which is to have a GUI frontend for borg, no command line experience needed. 

We could implement an alternative right click action for refresh that would do the slow archive size calculations.
If we do that, then we should do a similar think for the check, since there is a full integrity verification mode.

**Comment by rblenis** [comment]
I would be in favor of having a "Full Refresh" button that does the slow refresh, but brings all the cached data up-to-date. You could even give a warning that the operation may be slow compared to a normal "Refresh"
